packages/ui/src/components/va-button-toggle/VaButtonToggle.vue

```diff
@@ -42,7 +42,7 @@ export default defineComponent({
     options: { type: Array as PropType<any[]>, default: () => [] },
     color: { type: String as PropType<string>, default: 'primary' },
     textColor: { type: String as PropType<string>, default: undefined },
-    activeButtonTextColor: { type: String as PropType<string>, default: 'var(--va-white)' },
+    activeButtonTextColor: { type: String as PropType<string>, default: '#ffffff' },
```

Comment:
```suggestion
    activeButtonTextColor: { type: String as PropType<string>, default: 'white' },
```

We have white in ColorConfig's default theme, so getColor shoulw handle it 👍

Reply:
No, we haven't :)

Reply:
UPD: it was decided to add white & black in default theme colors preset.